src/plugins/one-letter-css.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @author denisx <github.com@denisx.com>
+ */
+
+const loaderUtils = require('loader-utils');
+
+export default class OneLetterCss {
+  constructor() {
+    // Save char symbol start positions
+    this.a = 'a'.charCodeAt(0);
+    this.A = 'A'.charCodeAt(0);
+    // file hashes cache
+    this.files = {};
```

Comment:
Here potential memory leak in watch mode, if will have a lot of renamed files, the object will grow

Reply:
fixed at last call - getStat()